torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -474,7 +597,7 @@ def perspective_image_tensor(
 
 def perspective_image_pil(
     img: PIL.Image.Image,
-    perspective_coeffs: float,
+    perspective_coeffs: List[float],
```

Comment:
I assume that was a bug/typo previously?

Reply:
yes, exactly